libnavigation-base/src/test/java/com/mapbox/navigation/base/internal/route/AnnotationsRefresherTest.kt

```diff
@@ -219,6 +219,8 @@ class AnnotationsRefresherTest(
                         .distance(listOf(1.2, 3.4, 5.6, 7.8, 9.0))
                         .duration(listOf(11.2, 33.4, 55.6, 77.8, 99.0))
                         .speed(listOf(41.0, 42.5, 43.1, 44.6, 45.9))
+                        .freeflowSpeed(listOf(1, 2, 3, 4, 5))
```

Comment:
Are you also gonna test it in instrumentation tests? I see a lot of room for error for NN team, they already said something would be wrong after refresh.

Reply:
Which test do you suggest?

Reply:
The same, but with NN involved. :)
We already have some tests that test annotations refresh, we could just extend them.